### PR TITLE
SiccarPoint/ErosionDeposition docs

### PR DESCRIPTION
**Pull request by SiccarPoint**
This PR adds some basic documentation to the ErosionDeposition component.

### CODE REVIEW

**Comment by SiccarPoint** [comment]
This PR partially addresses #792 (@cmshobe )

**Comment by SiccarPoint** [comment]
If one of you three could just scan what I wrote for any obvious boo boos...

File: landlab/components/erosion_deposition/erosion_deposition.py [hunk 1]
```diff
@@ -8,14 +8,42 @@
...
+      dimensionless number.
+
+    Erosion of the bed follows a stream power formulation, i.e.,
+
+    E = K * q ** m_sp * S ** n_sp - optional threshold (see above)
+
+    Note that in this implementation v_s becomes the parameter that
+    fundamentally controls response style. Very small v_s will lead to a
+    detachment-limited response style, very large v_s will lead to a
+    transport-limited response style. v_s == 1 means equal contributions from
```
Comment (gregtucker) [inline landlab/components/erosion_deposition/erosion_deposition.py]: I'm not sure about this interpretation of v_s = 1. You still have drainage area (or discharge), collectively forming a length scale.

Comment (SiccarPoint) [reply landlab/components/erosion_deposition/erosion_deposition.py]: Bump @cmshobe ... Good to get this in ASAP before it just atrophies.

Comment (cmshobe) [reply landlab/components/erosion_deposition/erosion_deposition.py]: @SiccarPoint I am so sorry!! On it now

Comment (cmshobe) [reply landlab/components/erosion_deposition/erosion_deposition.py]: @SiccarPoint @gregtucker:

The deal here is that the controlling dimensionless quantity is v_s/r, where r is the runoff rate from Q=rA. When v_s/r = 1, contributions from the DL and TL regimes are equal. When that ratio gets huge, we go to the TL regime, and vice versa. The reason I told Dan that v_s was the control is that in the Landlab implementation as far as I know we don't allow r to be a free parameter.

I'm going to fix the docs right now to make this clear.

**Comment by cmshobe** [comment]
I've superseded this with PR #836 which makes these changes and also the #795 fix

**Comment by SiccarPoint** [comment]
Cool. Killing this.

**Pull request closed without merging**